### PR TITLE
Record applicant's language in CSV export

### PR DESCRIPTION
**Pull request by hmhsu**
### Description
Record applicant's language in CSV export

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: universal-application-tool-0.0.1/app/services/export/ExporterService.java [hunk 1]
```diff
@@ -94,8 +94,7 @@ CsvExportConfig generateDefaultCsvConfig(long programId) {
 
     // Create a map from a key <block id, question index> to an answer with every application. It
     // doesn't matter which answer ends up in the map, as long as every <block id, question index>
-    // is
-    // accounted for.
+    // is accounted for.
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/services/export/ExporterService.java:97]: praise: thanks for fixing this formatting!

File: universal-application-tool-0.0.1/app/services/export/ExporterService.java [hunk 2]
```diff
@@ -131,6 +130,8 @@ private CsvExportConfig generateDefaultCsvConfig(ImmutableList<AnswerData> answe
     columnsBuilder.add(Column.builder().setHeader("ID").setColumnType(ColumnType.ID).build());
     columnsBuilder.add(
         Column.builder().setHeader("Submit time").setColumnType(ColumnType.SUBMIT_TIME).build());
+    columnsBuilder.add(
+        Column.builder().setHeader("Language").setColumnType(ColumnType.LANGUAGE).build());
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/services/export/ExporterService.java:134]: nit: maybe "Applicant Language" would make it clearer this is the applicant's preferred language? Either way is fine though!

**Review by carolinedanzi — APPROVED** [review]
Thanks for including this! Any tests we need to update? Otherwise looks good

File: universal-application-tool-0.0.1/app/services/export/ExporterService.java (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/services/export/ExporterService.java:97]: thanks!

File: universal-application-tool-0.0.1/app/services/export/ExporterService.java (hunk 2 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/services/export/ExporterService.java]: Good suggestion. done!

**Pull request merged**